cmd/hzpaste-api/main.go

```diff
@@ -71,9 +64,25 @@ func getFileName() string {
 	if len(env) == 0 {
 		env = "development"
 	}
-	filename := []string{"internal/config/", "config.", env, ".json"}
-	_, dirname, _, _ := runtime.Caller(0)
-	filePath := path.Join(filepath.Dir(dirname), strings.Join(filename, ""))
+	filename := []string{"config/", "config.", env, ".json"}
```

Comment:
Configuration file resides in the "internal/config" path

Reply:
That makes sense, but still, after you get a compiled binary you'll have to put configuration JSON to exact the same directory which sometimes could not be possible if you transfer binary to the remote host.
And it compiled in this branch without env vars

Reply:
Got your point (I forget about it 😮‍💨 ) - hmm, is it reasonable to create `.config.dev.json` in the same directory?

Reply:
I guess that we should move that config to examples and point to it in the documentation.
'make' targets should be specified with env vars I think